spec/helpers/simplified_admin_set_selection_work_form_helper_spec.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe HykuAddons::SimplifiedAdminSetSelectionWorkFormHelper do
+  include Devise::Test::ControllerHelpers
+
+  let(:user) { create(:user) }
+  let(:model_class) do
+    Class.new(GenericWork) do
+      include ::HykuAddons::GenericWorkOverrides
+    end
+  end
+  let(:work) { model_class.new(title: ["Moomin"], depositor: user.user_key) }
+  let(:form_class) do
+    Class.new(Hyrax::GenericWorkForm) do
+      include ::HykuAddons::GenericWorkFormOverrides
+    end
+  end
+  let(:form) { form_class.new(work, nil, nil) }
```

Comment:
I think this could be simplified since it is super complex in `hyrax-doi`.
```suggestion
  let(:work) { GenericWork.new(title: ["Moomin"], depositor: user.user_key) }
  let(:form) { Hyrax::GenericWorkForm.new(work, nil, nil) }
```